### DISCUSSION TITLE
Report for violent!

### DISCUSSION THREAD

**Discussion by imfluxxy** [community/community#48253] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi Github,
I'm a user from github, i'd send this feedback for request give back my repository back, someone maybe send to RDP_Action repository
i'm the owner of that repository! that repository doesn't have violence, so please give back thank you

-Nastionzie175

**Comment by vedantmgoyal9** [comment, marked as answer]
You need to create a support ticket for it.

**Reply by queenofcorgis** [reply]
@vedantmgoyal2009 is correct - the community is unable to assist in these matters. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them - I'll be locking this post to prevent any confusion :)
